www/js/lib/uiUtil.js

```diff
@@ -32,6 +32,48 @@ if (webpMachine) {
 }
 
 define(rqDef, function(settingsStore) {
+
+    /**
+     * Displays a Bootstrap alert or confirm dialog box depending on the options provided
+     * 
+     * @param {String} message The alert message to display in the body of the modal 
+     * @param {String} label The modal's label or title which appears in the header (optional, Default = "Confirmation" or "Message")
+     * @param {Boolean} isConfirm If true, the modal will be a confirm dialog box, otherwise it will be a simple alert message 
+     * @param {String} declineConfirmButton The text to display on the decline confirmation button (optional, Default = "Cancel") 
+     * @param {String} approveConfirmButton  The text to display on the approve confirmation button (optional, Default = "Confirm")
+     * @param {String} closeMessageButton  The text to display on the close alert message button (optional, Default = "Okay")
+     * @returns {Promise<Boolean>} A promise which resolves to true if the user clicked Confirm, false if the user clicked Cancel/Okay, backdrop or the cross(x) button
+     */
+    function systemAlert(message, label, isConfirm, declineConfirmButton, approveConfirmButton, closeMessageButton) {
+        declineConfirmButton = declineConfirmButton || document.getElementById("declineConfirm").innerText;
```

Comment:
You should use `.textContent` in these 3 lines, instead of `.innerText`.
First to be consistent with lines 55 to 57.
And also (but it's minor) because innerText is supported by Firefox >=45 and latest FirefoxOS is using the Gecko of Firefox 44

After fixing that, the new popups seem to work well on my Firefox OS device, even with its small screen.